.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+lint-staged
```

Comment:
Fyi, this should be `yarn lint-staged` as you're referencing a globally installed `lint-staged` in this case.

Reply:
True, I'll look into this today/tomorrow morning 👍

Reply:
No worries. I actually drafted a PR #94 for this.